jetson/odrive_bridge/src/__main__.py

```diff
@@ -83,18 +90,14 @@ def lcmThreaderMan():
         start_time = t.clock()
         try:
             publish_encoder_msg()
-        except NameError:
-            pass
-        except AttributeError:
-            pass
-        except Exception:
+        except (NameError, AttributeError, fibre.protocol.ChannelBrokenException):
             pass
 
 
 events = ["disconnected odrive", "disarm cmd", "arm cmd", "odrive error"]

```

Comment:
You should make enums for the events and the states

Reply:
enums are annoying in python, this is the work around that's about as efficient/readable

Reply:
https://docs.python.org/3/library/enum.html 
They don't seem that annoying. It looks like you just create a class and reference it throughout. Why do you say they are annoying?

Reply:
would be annoying to get working in this code more so, i'd be adding a class within a class to call a function that acts the same as .str which regardless is in there

Reply:
i think could be rewritten to be if self.state == ArmState no casting at all, not sure how python will handle the comparison though would have to test

Reply:
Gotcha. I didn't realize that self.state was a class and had functionality beyond just being a variable that you are comparing against

Reply:
Seems like you could make a getter for interfacing with the class, but what you have is clear enough